### PR TITLE
issue#4 | @rebeccahongsf | update ordering to work on multiple browsers

### PR DESCRIPTION
**Pull request by rebeccahongsf**
## Summary
- update to descending order which works on multiple browsers

## Issue ticket number and link
- #4 

## Checklist before requesting a review
- [x] I have performed a self-review of my code

## Screenshots (if applicable)

Firefox:
image

Chrome:
image

### CODE REVIEW

**Comment by rebeccahongsf** [comment]
@jwu910

**Review by jwu910 — APPROVED** [review]

File: scripts.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ $(document).ready(() => {
 
       // Loop through every category and write mentors to the page
       Object.entries(categoryMentors)
-        .sort((a, b) => a[1].length < b[1].length)
```
Comment (jwu910) [inline scripts.js:74]: I think we would like descending order. should this be flipped to b - a instead?

**Review by jwu910 — CHANGES_REQUESTED** [review]

File: scripts.js (hunk 1 above)
Comment (rebeccahongsf) [reply scripts.js:74]: Sounds good! I've updated the PR and screenshots. Ready for another round of review @jwu910

**Review by jwu910 — APPROVED** [review]

**Comment by jwu910** [comment]
Thanks! Merged

**Pull request merged**